.github/workflow-samples/groovy-dsl/settings.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 
 gradleEnterprise {
     buildScan {
-        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceUrl = "https://gradle.com/help/legal-terms-of-use"
```

Comment:
It redirects to `https://gradle.com/legal/terms-of-use/`, is this really the one we should use?

Reply:
https://gradle.slack.com/archives/C061EQ3MT1D/p1710172339902639